shared/src/db/database.ts

```diff
@@ -84,6 +84,7 @@ class Database extends Dexie {
             this.accessMapRef,
             () => {
                 this.deleteRevoked();
+                this.deleteRevoked(true); // Remove change documents
```

Comment:
Not a big fan of these boolean arguments, as reading the code makes it not immediately clear why the same method is called twice. Some things that could improve it:
- Prefix it with the argument name: `this.deleteRevoked(changeDocs: true)`
- Make separate methods that call `deleteRevoked`, like `deleteRevokedNormalDocs` and `deleteRevokedChangeDocs`